src/NI-SCOPE Time Domain/README.md

```diff
@@ -0,0 +1,30 @@
+## NI-SCOPE Time Domain Measurement 
+
+This example performs time domain measurement using NI SCOPE.
+
+
+### Features
+
+- Uses the NI-SCOPE LabVIEW API.
+- Pin-aware, supporting one session and one pin.
+  - Uses the same selected measurement function and range for all selected pin/site combinations.
+- Includes InstrumentStudio project files.
+- Includes a TestStand sequence showing how to configure the pin map, register.
```

Comment:
remove the period at the end of this line, and then lines 12 and 13 should probably be on one line.